### PR TITLE
optimized version of hls_to_rgb() and rgb_to_hls()

### PR DESCRIPTION
**Pull request by lferraz**
### Description

in colab I got this speed up (the ones with the _ are the optimized ones):

Screenshot 2021-07-11 at 16 05 48

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### PR Implementer
This is a small checklist for the implementation details of this PR.

If there are any questions regarding code style or other conventions check out our
[summary](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst).

- [ ] Did you discuss the functionality or any breaking changes before ?
- [ ] **Pass all tests**: did you test in local ? `make test`
- [ ] Unittests: did you add tests for your new functionality ?
- [ ] Documentations: did you build documentation ? `make build-docs`
- [ ] Implementation: is your code well commented and follow conventions ? `make lint`
- [ ] Docstrings & Typing: has your code documentation and typing ? `make mypy`
- [ ] Update notebooks & documentation if necessary

### KorniaTeam
<details>
  <summary>KorniaTeam workflow</summary>

  - [ ] Assign correct label
  - [ ] Assign PR to a reviewer
  - [ ] Does this PR close an Issue? (add `closes #IssueNumber` at the bottom if
        not already in description)

</details>

### Reviewer
<details>
  <summary>Reviewer workflow</summary>

  - [ ] Do all tests pass? (Unittests, Typing, Linting, Documentation, Environment)
  - [ ] Does the implementation follow `kornia` design conventions?
  - [ ] Is the documentation complete enough ?
  - [ ] Are the tests covering simple and corner cases ?

</details>

### CODE REVIEW

File: kornia/color/hls.py [hunk 1]
```diff
@@ -83,8 +88,8 @@ def rgb_to_hls(image: torch.Tensor, eps: float = 1e-8) -> torch.Tensor:
 
     if image.requires_grad:
         return torch.stack([h, l, s], dim=-3)
-    else:
-        return image_hls
+
+    return image_hls
```
Comment (edgarriba) [inline kornia/color/hls.py:92]: @lferraz can we return this before ?

Comment (lferraz) [reply kornia/color/hls.py:95]: before?

**Review by edgarriba — APPROVED** [review]

**Review by edgarriba — APPROVED** [review]
